rust/scripts/bench.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+mkdir -p ~/benches
+
+cd Tosca
+git fetch --all
+
+RUNS=20
+TIMEOUT=1h
+REF_BRANCH="origin/main"
```

Comment:
note: because you checkout this, this could be a commit as well.  or a tag. any git ref would do